frappe/public/js/frappe/form/footer/timeline.js

```diff
@@ -547,10 +547,14 @@ frappe.ui.form.Timeline = class Timeline {
 			log.color = 'dark';
 			log.sender = log.owner;
 			log.comment_type = 'Milestone';
-			log.content = __('{0} changed {1} to {2}', [
-				frappe.user.full_name(log.owner).bold(),
-				frappe.meta.get_label(this.frm.doctype, log.track_field),
-				log.value.bold()]);
+			log.content = __(
```

Comment:
Translation strings should be in one line.

https://frappeframework.com/docs/user/en/translations#3-blocks

Reply:
> This PR is trying to fix the issue with images in comments, but it also touches unrelated lines like the translation strings. Ideally, the changes in the commit should reflect the description.

@netchampfaris 
I actually had no intention to touch any translation string.
I had only edited the image parts
But "Frappe Linter / Translation Syntax Check" was failing at all these places 
So had to make these changes to make the linter happy

Reply:
Done